solution/text-extractor/extractors/binary.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+from olefile import OleFileIO as ofio
+
+from .extractor import Extractor
+
+
+class BinaryExtractor(Extractor):
```

Comment:
Binary extractor is the name for now.  This solution will probably only work for .doc files in the end though.